File: Solutions/442. Find All Duplicates in an Array/442. Find All Duplicates in an Array.py
class Solution:
    def findDuplicates(self, nums: List[int]) -> List[int]:
        cnt=[0]*(len(nums)+1)
        res=[]
        for n in nums:
            cnt[n]+=1
        for i in range(1,len(nums)+1):
            if cnt[i]==2:
                res.append(i)
        return res

File: Solutions/442. Find All Duplicates in an Array/442. Find All Duplicates in an Array.go
func findDuplicates(nums []int) []int {
    cnt:=make([]int,len(nums)+1)
    var res []int
    for _,n:=range(nums){
        cnt[n]+=1
    }
    for i:=1;i<=len(nums);i+=1{
        if cnt[i]==2{
            res=append(res,i)
        }
    }
    return res
}